README.md

```diff
@@ -108,85 +155,239 @@ in the case of lists sets and scalars, rows with key and value text columns
 for hashes, and rows with a value text columns and an optional numeric score
 column for zsets.
 
-The following parameter can be set on a user mapping for a Redis
-foreign server:
+## IMPORT FOREIGN SCHEMA options
 
-password:	The password to authenticate to the Redis server with.
-     Default: <none>
+`redis_fdw` **doesn's support** [IMPORT FOREIGN SCHEMA](https://www.postgresql.org/docs/current/sql-importforeignschema.html) and accepts no custom options for this command. There is no formal storing schema in Redis in oppose to RDBMS.
 
-Insert, Update and Delete
--------------------------
+## TRUNCATE support
 
-PostgreSQL acquired support for modifying foreign tables in release 9.3, and
-now the Redis Foreign Data Wrapper supports these too, for 9.3 and later
-PostgreSQL releases. There are a few restriction on this:
+`redis_fdw` don't implements the foreign data wrapper `TRUNCATE` API, available
```

Comment:
s/don't/doesn't/

Reply:
Fixed in https://github.com/pg-redis-fdw/redis_fdw/pull/37/commits/01bce9dfdce3ebcfe0dc6dc5e5ad05abede8315a, please verify.